src/index.js

```diff
@@ -0,0 +1,9 @@
+import './styles/reset.css';
+import './styles/common.css';
+
+import { Layout } from './components/Layout';
+import { MovieListView } from './view/movieListView';
+
+const app = document.getElementById('app');
+app.appendChild(Layout());
+app.addEventListener('DOMContentLoaded', MovieListView());
```

Comment:
이 부분이 무척 이상하네요 ㅎㅎ

MoveListView는 함수이고, 반환값은 undefined 입니다.
즉, 이 코드는 DOMContentLoaded 가 발생하는 시점에 실행되는 함수가 아니라
그냥 함수를 실행해서 undefined를 addEventListener에게 반환하는 형태입니다.

의도한 동작과는 아예 다른거죠.

```suggestion
app.addEventListener('DOMContentLoaded', MovieListView);
```

그래서 이렇게 해야 하지 않을까요?